libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3601,7 +3601,9 @@ MAV_RESULT GCS_MAVLINK::handle_command_preflight_can(const mavlink_command_long_
                 // TODO - Run PiccoloCAN pre-flight checks here
                 break;
             case AP_BoardConfig_CAN::Protocol_Type_UAVCAN:
+                break;
```

Comment:
These should not be required and just add verbiage.  The compilers make a carve-out for fall-throughs where there's no statements between the case labels.

That being said, removing the default: here would be nice if we handle all the cases!

Reply:
Sure, I will clean this up.